bin/pycbc_live

```diff
@@ -314,8 +317,11 @@ class LiveEventManager(object):
 
             if single is None:
                 continue
+
+            other_ifos = [i for i in active if i is not ifo]
```

Comment:
Use set operations here, they are cool. Grep for `set(` for examples in `pycbc_live`.

Reply:
I've looked and implementation of set operations in this case looks tricky, and degrades readability (and efficiency) of the code quite a bit, mainly as the ifos are not already available as a list at this point

I think that leaving it without set operations makes sense in this case